Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM heroku/cedar:14
+FROM heroku/heroku:16
 RUN curl "https://github.com/gliderlabs/herokuish/releases/download/v0.3.32/herokuish_0.3.32_linux_x86_64.tgz" \
 		--silent -L | tar -xzC /bin
-RUN /bin/herokuish buildpack install \
```

Comment:
Because herokuish is meant to provide heroku bp deployment compatibility, this should not be removed.